src/routes/login.svelte

```diff
@@ -131,6 +154,15 @@
         have a keyfile, you can get one by creating an <a
           href="https://www.arweave.org/wallet">Arweave Wallet</a>.
       </p>
+      <div class="weaveid-login">
```

Comment:
```suggestion
      <hr />
      <div class="weaveid-login">
```
I think we need some sort of differentiator so users know they can drag/drop their keyfile *or* login with WeaveID.

Reply:
What do you think about this?
![image](https://user-images.githubusercontent.com/9200820/106580989-a6771100-6542-11eb-8927-b1dc2a7593f7.png)

Reply:
@t8 The error you mentioned has been fixed.

Reply:
This looks great! Thanks for addressing this.